cost/volumes/uav_policy.pt

```diff
@@ -0,0 +1,288 @@
+﻿# TO-DO: 
+#	Add support for deleting old unattached volumes.
+#
+
+name "Unattached Volumes"
+rs_pt_ver 20180301
+short_description "Finds unattached volumes older than specified number of days and reports them."
+severity "medium"
+category "Cost"
+
+permission "general_permissions" do
+  resources "rs_cm.volumes", "rs_cm.audit_entries", "rs_cm.placement_groups", "rs_cm.clouds", "rs_cm.tags", "rs_cm.deployments"
+  actions   "rs_cm.*"
+end
+
+parameter "param_email" do
+  type "string"
+  label "Email address to send escalation emails to"
+  allowed_pattern /[a-zA-Z0-9_\-\.]+@[a-zA-Z0-9_\-\.]/
+  min_length 1
+  constraint_description "Must be an email address."
+end
+
+parameter "param_days_old" do
+  type "number"
+  label "Identify volumes that have been unattached for the given number of days."
+  default 30
+end
+
+parameter "param_delete" do
+  type "string"
+  label "Enter the word, DELETE (all UPPERCASE), to have identified volumes automatically deleted."
+  allowed_values "ALERT ONLY", "ALERT AND DELETE"
```

Comment:
Instead of **alert**, should it be **create incident** only?  alert may be confused with CM alerts.

Reply:
Good point. 
I changed it to "EMAIL ONLY" and "EMAIL AND DELETE"